src/Ledger/GovernanceActions.lagda

```diff
@@ -61,7 +90,37 @@ data GovAction : Set where
   ChangePParams    : UpdateT → PPHash                 → GovAction
   TreasuryWdrl     : (Credential ⇀ Coin)              → GovAction
   Info             :                                    GovAction
-
+\end{code}
+\caption{Governance actions}
+\label{defs:governance}
+\end{figure*}
+\begin{table}[h]
+\begin{longtable}[]{@{}
```

Comment:
I think this table would look a lot better if the constructor name would go on the left (still under the `Action` heading) and the `Agda type` heading was removed. I don't think we need short and long descriptions of every action in a table.